docs/content/0.1.0/docs/dj-concepts/nodes.rst

```diff
@@ -2,203 +2,170 @@
 weight: 10
 ---
 
-.. _functions:
+-----
+Nodes
+-----
 
----------
-Functions
----------
+In DJ, nodes play a central role. Understanding the relationships between nodes is key to understanding how DJ works.
+All node types are similar in many ways. Let’s start by covering their similarities.
 
-Currently, DJ supports only a small subset of SQL functions, limiting the definition of metrics. In order to add new functions to DJ we need to implement two things:
+Similarities Between Node Types
+-------------------------------
 
-1. Type inference for the function. This is easier in some functions and harder in others. For example, the ``COUNT`` function always return an integer, so its return type is the same regardless of the input arguments. The ``MAX`` function, on the other hands, return a value with the same type as the input argument.
+A summary of things that are true of all nodes:
 
-2. Translation to SQLAlchemy. In order to run queries, DJ parses the metric definitions (written in ANSI SQL), and converts them to a SQLAlchemy query object, so it can be translated to different dialects (Hive, Trino, Postgres, etc.). Some functions are easier to translate, specially if they are already defined in ``sqlalchemy.sql.functions``. Others, like ``DATE_TRUNC``, are more complex because they are dialect specific.
+* All nodes have a name and a description
+* All nodes have a schema defined as named columns, each with a specific type
+* All nodes have a system defined state of either valid or invalid
+* All nodes have a user-defined mode of either draft or published
+* All nodes track the parent nodes they depend on and the children nodes that depend on them
```

Comment:
> and the children nodes that depend

If I am reading our data model right, then above is just a side effect of the tracking of the parents. Not sure we need to say it here.